spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/channel/PostgresChannelMessageTableSubscriber.java

```diff
@@ -298,7 +299,9 @@ public boolean isRunning() {
 	}
 
 	private static String getKey(Object input) {
-		return UUIDConverter.getUUID(input).toString();
+		UUID uuid = UUIDConverter.getUUID(input);
+		Assert.notNull(uuid, "UUID must not be null");
+		return uuid.toString();
```

Comment:
I think after `@Contract("null -> null; !null -> !null")` in the `UUIDConverter`, we won't need these changes here.